interface/therapy_groups/therapy_groups_models/users_model.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * interface/therapy_groups/therapy_groups_models/users_model.php contains the model for users.
+ *
+ * This model fetches the users data to be used for the therapy group from the DB.
+ *
+ * Copyright (C) 2016 Shachar&Amiel <shachar058@gmail.com> <amielboim@gmail.com>
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * @author  Shachar Zilbershlag <shachar058@gmail.com>
+ * @author  Amiel Elboim <amielboim@gmail.com>
+ * @link    http://www.open-emr.org
+ */
+
+class Users{
+
+    const TABLE = 'users';
+    const EVENTS_TABLE = 'openemr_postcalendar_events';
+
+    /**
+     * Get all users' ids and full names from users table.
+     * @return array
+     */
+    public function getAllUsers(){
+
+        $sql = 'SELECT id, fname, lname FROM ' . self::TABLE . ' WHERE active = 1';
+
+        $users = array();
+        $result = sqlStatement($sql);
+        while($u = sqlFetchArray($result)){
+            $users[] = $u;
+        }
+
+        return $users;
+    }
+
+    /**
+     * Get user name by user id from users table.
+     * @param $uid
+     * @return string
+     */
+    public function getUserNameById($uid){
+        $sql = 'SELECT fname, lname FROM ' . self::TABLE . ' WHERE id = ?';
+
+        $result = sqlStatement($sql, array($uid));
```

Comment:
I say this because:
1. Is more clear codewise.
2. There is a developer whom is working on converting the database engine to PDO, and this will make it easier.